omnibus/config/software/chef-dk.rb

```diff
@@ -79,23 +79,14 @@
   appbundle "test-kitchen", lockdir: project_dir, gem: "test-kitchen", without: %w{changelog debug docs provisioning}, env: env
   appbundle "inspec", lockdir: project_dir, gem: "inspec", without: %w{deploy tools maintenance integration}, env: env
 
-  %w{chef-apply chef-dk chef-vault ohai opscode-pushy-client cookstyle dco berkshelf}.each do |gem|
+  %w{chef-dk chef-apply chef-vault ohai opscode-pushy-client cookstyle dco berkshelf}.each do |gem|
```

Comment:
Was there a specific reason for the order change here? I think we were trying to keep workstation stuff at the front for code organization.

Reply:
It was just a sync from DK where it's this order